acados/ocp_qp/ocp_qp_partial_condensing.c

```diff
@@ -160,20 +167,26 @@ int ocp_qp_partial_condensing_opts_calculate_size(void *dims_)
 
     int N = dims->orig_dims->N;
 
+	// populate dimensions of reduced qp
+	d_ocp_qp_dim_reduce_eq_dof(dims->orig_dims, dims->red_dims);
```

Comment:
These routines remove easy equality constraints form the QP formulation, i.e. "box" equality constraints on the degrees of freedom (DOF). Therefore, the QP is "reduced" to a smaller one.
Since this is not "condensing" and that name is taken anyway, I went for another name: reduce/restore instead of condense/expand.